tapir/accounts/emails/create_account_reminder_email.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+
+from django.utils.translation import gettext_lazy as _
+
+from tapir.coop.models import ShareOwner
+from tapir.core.tapir_email_base import TapirEmailBase
+
+
+class CreateAccountReminderEmail(TapirEmailBase):
+    def __init__(self, share_owner: ShareOwner):
+        super().__init__()
+        self.draft_user = share_owner
+
+    @classmethod
+    def get_unique_id(cls) -> str:
+        return "tapir.accounts.create_account_reminder"
+
+    @classmethod
+    def get_name(cls) -> str:
+        return _("Create account reminder")
+
+    @classmethod
+    def get_description(cls) -> str:
+        return _(
+            "Sent to active member if they haven't created the account 1 month after becoming member."
+        )
+
+    def get_subject_templates(self) -> List:
+        return [
+            "accounts/email/create_account_reminder.subject.html",
+            "accounts/email/create_account_reminder.subject.default.html",
+        ]
+
+    def get_body_templates(self) -> List:
+        return [
+            "accounts/email/create_account_reminder.body.html",
+            "accounts/email/create_account_reminder.body.default.html",
+        ]
+
+    @classmethod
+    def get_dummy_version(cls) -> TapirEmailBase | None:
+        share_owner = ShareOwner.objects.filter(user__isnull=True).order_by("?").first()
```

Comment:
I'm not sure about this part - `user__isnull=True` is the opposite what we want to check when sending this email, but maybe for dummy email it's fine like that.

Reply:
`user__isnull=True` looks correct to me? Since we want a member without account, we want to user field to be Null?